src/tasks/LargeFileUploadTask.ts

```diff
@@ -204,13 +219,14 @@ export class LargeFileUploadTask {
 	}
 
 	/**
+	 * @deprecated This function has been moved into FileObject interface.
 	 * @public
 	 * Slices the file content to the given range
 	 * @param {Range} range - The range value
 	 * @returns The sliced ArrayBuffer or Blob
 	 */
 	public sliceFile(range: Range): ArrayBuffer | Blob {
```

Comment:
console warn maybe? so people not paying attention to compile time warning and/or not using typescript get a message in the console

Reply:
Done